src/stservo/gui/servo_gui.py

```
@@ -1544,6 +1701,280 @@ def clear_sync_results(self):
+                
+                # Unlock EPROM first
+                self.packet_handler.unLockEprom(self.current_servo_id)
+                self.log_message(f"Unlocked EPROM for servo ID {self.current_servo_id}")
+            
+            # Write value
+            if reg_info['size'] == 1:
+                comm_result, error = self.packet_handler.write1ByteTxRx(self.current_servo_id, self.selected_register_addr, new_value)
+            else:
+                comm_result, error = self.packet_handler.write2ByteTxRx(self.current_servo_id, self.selected_register_addr, new_value)
+            
+            if comm_result == COMM_SUCCESS and error == 0:
```

Ruff (0.12.2)

`COMM_SUCCESS` may be undefined, or defined from star imports

(F405)